numpy/core/_methods.py

```diff
@@ -213,10 +228,10 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     else:
         x = um.multiply(x, um.conjugate(x), out=x).real
 
-    ret = umr_sum(x, axis, dtype, out, keepdims)
+    ret = umr_sum(x, axis, dtype, out, keepdims=keepdims, where=where)
```

Comment:
My recommendation would be to add
```
    if not keepdims:
        rcount = rcount.squeeze(axis=axis)
```
here

Reply:
Ugh, I see this doesn't work either.

Reply:
So should I maybe leave it as is?

Reply:
Probably best